corehq/apps/domain/calculations.py

```diff
@@ -377,18 +377,14 @@ def calced_props(dom, id, all_stats):
     }
 
 
-def total_distinct_users(domains=None):
+def total_distinct_users(domain):
```

Comment:
Actually it did work because `.domain` can accept a list or a string, but agreed that that is not obvious